README.md

```diff
@@ -1,4 +1,4 @@
-# Slugify
+# Slugify :abc:
```

Comment:
Github doesn't support the :<text>: syntax for emojis. Instead we need to use the emoji itself.
```suggestion
# Slugify 🔤

```